scripts/gulp/live-reload-server.js

```diff
@@ -99,6 +99,10 @@ function LiveReloadServer(port, config) {
 
                 // Open the sidebar when the page loads
                 openSidebar: true,
+                // Disable the click on the guest page outerside of the elements
```

Comment:
This just sets defaults if the page config doesn't override, I worry upstream might conflict needlessly. You have a better feel for this than me though, so just a note.